packages/components/src/Button.tsx

```diff
@@ -36,35 +36,33 @@ const button = cva(styles.base, {
 	},
 });
 
-const ButtonContext = createContext<ButtonProps | null>(null);
-
 interface ButtonVariants extends VariantProps<typeof button> {}
 interface ButtonProps extends AriaButtonProps, ButtonVariants {
 	ref?: Ref<HTMLButtonElement>;
 }
 
+const ButtonContext = createContext<ContextValue<ButtonProps, HTMLButtonElement>>(null);
```

Comment:
Not sure if that rings a bell, but when testing this yesterday, `isPressed` (from their example) was causing a TypeScript error as if this version didn't support it? 🤔